models/iaf_psc_exp.h

```diff
@@ -108,21 +158,32 @@ Parameters
 
 The following parameters can be set in the status dictionary.
 
-===========  =======  ========================================================
- E_L          mV      Resting membrane potential
- C_m          pF      Capacity of the membrane
- tau_m        ms      Membrane time constant
- tau_syn_ex   ms      Exponential decay time constant of excitatory synaptic
-                      current kernel
- tau_syn_in   ms      Exponential decay time constant of inhibitory synaptic
-                      current kernel
- t_ref        ms      Duration of refractory period (V_m = V_reset)
- V_m          mV      Membrane potential in mV
- V_th         mV      Spike threshold in mV
- V_reset      mV      Reset membrane potential after a spike
- I_e          pA      Constant input current
- t_spike      ms      Point in time of last spike
-===========  =======  ========================================================
+
+=============== ================== =============================== ========================================================================
+**Parameter**   **Default**        **Math equivalent**             **Description**
+=============== ================== =============================== ========================================================================
+``E_L``         -70 mV             :math:`E_\text{L}`              Resting membrane potential
+``C_m``         250 pF             :math:`C_{\text{m}}`            Capacity of the membrane
+``tau_m``       10 ms              :math:`\tau_{\text{m}}`         Membrane time constant
+``t_ref``       2 ms               :math:`t_{\text{ref}}`          Duration of refractory period
+``V_th``        -55 mV             :math:`V_{\text{th}}`           Spike threshold
+``V_reset``     -70 mV             :math:`V_{\text{reset}}`        Reset potential of the membrane
+``tau_syn_ex``  2 ms               :math:`\tau_{\text{syn, ex}}`   Rise time of the excitatory synaptic alpha function
+``tau_syn_in``  2 ms               :math:`\tau_{\text{syn, in}}`   Rise time of the inhibitory synaptic alpha function
+``I_e``         0 pA               :math:`I_\text{e}`              Constant input current
+``V_min``       :math:`-\infty` mV :math:`V_{\text{min}}`          Absolute lower value for the membrane potential
```

Comment:
A bit surprising, but `iaf_psc_exp` does not seem to have `V_min`. Adding it would be a different issue.
```suggestion
```